SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -66,8 +68,12 @@ mount_volatile_persistent_volume() {
     else
         # Check if /etc/mdadm.conf exists.
         if [ -f "/etc/mdadm.conf" ]; then
-            mdadm --assemble ${_volume} || \
-                die "Failed to assemble RAID volume."
+            # If mdadm.conf exists, assume still need to manage RAID arrays.
+            if mdadm --detail "${_volume}" &> /dev/null; then
```

Comment:
In order to make sure the RAID array works as we expect, I still add an additional logic checking it then help prepare it.

Reply:
Have you tried using `rd.md.uuid` combined with ` wait_for_dev <RAID DEV>`? My expectation is that dracut should assemble the raid array automatically, and `wait_for_dev` should block the mounts from running until its ready.

Reply:
We don't currently support adding kernel argument like `rd.md.uuid`. The `volume` here is passed through MIC / Trident API. 

In addition, the reason why we still need to put raid assemble logic here is that in the real world, we still see related error occurs without manually assemble raid. 

However, in this PR, specifically we just need an additional if check statement layer, checking if the `volume` is a raid array. Because sometimes, there is a combination case that `mdadm.conf` exists due to verity / verity-hash volumes are raid, but overlay persistent volume is a regular gpt partition.